basic/07b_robot_6_dof_arm.py

```diff
@@ -109,7 +111,13 @@ def execute_trajectory(group, model, trajectory, feedback):
 group.get_next_feedback(reuse_fbk=feedback)
 waypoints[:, 0] = feedback.position
 waypoints[:, 1] = joint_targets[:, 0]
+print("joint targets")
```

Comment:
These print statements are extraneous and should be removed for the public facing examples.